lib/helpers/rule-test-harness.js

```diff
@@ -101,48 +101,55 @@ module.exports = function generateRuleTests({
     let DISABLE_ONE = `{{! template-lint-disable ${name} }}`;
     let DISABLE_ONE_LONG = `{{!-- template-lint-disable ${name} --}}`;
 
-    let linter;
+    function copyPlugins() {
+      let pluginsCopy;
+      if (plugins && plugins.length > 0) {
+        pluginsCopy = plugins.map(({ configurations, name, rules }) => {
+          let configurationsCopy;
+          if (configurations) {
+            configurationsCopy = JSON.parse(JSON.stringify(configurations));
```

Comment:
Copying like this won't work if a RegExp is passed as a rule config value. Would that be a problem?